### DISCUSSION TITLE
Issues transferring domain

### DISCUSSION THREAD

**Discussion by notdotscott** [vercel/community#4655] [Help] [answered]
### Summary

I've unlocked my domain with my old registrar and begun the transfer to Vercel using my EPP code. However, the transfer seems to be stuck and Vercel keep emailing to say that I need to authorise the transfer with my old registrar - my old registrar say there's nothing they need to do. Can someone help me?

### Steps to Reproduce

None, this is a domain transfer issue but I can't create a ticket for it as I'm only on a Hobby plan (despite paying Vercel $50 for the domain renewal).

**Comment by amyegan** [comment, marked as answer]
Sorry you ran into trouble with your domain transfer, @notdotscott. Please [submit a case](https://vercel.com/help#issues) to the Customer Success Team under the `Billing` problem area.
